### ISSUE TITLE
Transpose export option for LQAS data

### ISSUE THREAD

**Issue by bnygren** [xlg8/APMIS-Project#712]
**Situation**: Users make use of LQAS data in long format (numbered by house).  They currently format outside of APMIS. 

**Solution**: In campaign data, LQAS data should have a button for exporting as long format (transposed). See attachment for more detail.

**File name:** File name should follow the existing non-transposed export format, with the word long appended as shown below:
- CampaignName_LQAS Cluster Data Collection Form_LONG_YYYYDDMM

**Comma handling** - update per discussion: 
- Commas in labels can be replaced with spaces to prevent formatting/parsing issues with CSV output

To align the wide and long formats please rename the following in long:

labels in long format:
Dcode should be -> DCode
Creating User -> Creating user
Verified -> Verified Status
Published -> Published Status

Variables in long format:
rCode -> rcode
pCode -> pcode
cCode -> ccode
isVerified -> isverified
isPublished -> ispublished
cluster -> Cluster

Preferred order of variables for transposed data per users:

| LABEL | VARIABLE |
|---|---|
| Campaign | campaign |
|Form | form |
|Region | region |
|RCode | rcode |
|Province | 		province |
|PCode	 | 	pcode |
|District	 | 	district |
|DCode	 | 	dcode |
|Cluster	 | 	Cluster |
|Cluster Number	 | clusterNumber |
|CCode		 | ccode |
|Form Phase	 | formType |
|Source		 | source |
|Creating user	 | creatingUser |
|Date		 | VisitDate |
|Name of Surveyor	 | Surveyor |
|Lot Number	 | LotNo |
|Lot Cluster Number	 | LotClusterNo |
|Village Name	 | Village |
|Village Code (Ten Digits)	 | villagecode |
|Household Number	 | housenumber |
|HouseTotalChildrenSeen	 | houseTotalChildrenSeen |
|Age of selected child in months (60-119)	 | childrenAge |
|Gender		 | Gender |
|Is finger marked for OPV?	 | FM |
|Reason if finger is not marked	 | Reasons |
|If any area/household are not visited, please write down the location and reasons	 | Comment |
|Verified Status	 | isverified |
|Published Status	 | ispublished |

**Comment by bnygren** [comment]
per @DevSeg one solution maybe adding prefixes to field names to allow for IDing the fields to transpose. Will also need to account for historical data. 
We will revisit at next dev meeting

**Comment by bnygren** [comment]
Please see attached comparison in issue description of export data long format against wide with a few issues noted.

**Comment by bnygren** [comment]
Getting some mismatched translation on the SEP TEST campaign but not others:
![Image](https://github.com/user-attachments/assets/bc5fbf2d-6a17-49ca-9be3-84786c8886e9)

![Image](https://github.com/user-attachments/assets/d83946b6-2106-40b2-ab65-c8a7f08c977c)

**Comment by bnygren** [comment]
@DevSeg @omoluabidotcom Final Col (AF) in the transposed exports has no headers but has a 1 in the third row. Can we explain this? Thanks.

**Comment by bnygren** [comment]
Waiting on feedback and resolution on the above comments prior to closing

**Comment by bnygren** [comment]
Hi @DevSeg @Eutit5 I have done a detailed review, compared against the wide format variables and labels for consistency and attached an updated spreadsheet with a few clarifications.

**Comment by bnygren** [comment]
@DevSeg Please see the updated minor spelling changes in the main description to align wide and long

**Comment by bnygren** [comment]
The issue description was updated with some changes to the field names and order, hence being tagged as not complete, rather than create a new issue for these minor changes. Thanks

**Comment by bnygren** [comment]
verified
